content/getting-started/cli.md

```diff
@@ -244,8 +246,41 @@ Whichever address and port it says there is the one to open in the browser.
 
 Once open, you'll be presented with the login screen. Paste in the web login
 code you copied from the dojo in the previous step and hit "continue". You'll
-now be taken to your homescreen, with tiles for the default apps such as Groups
-and Terminal.
+now be taken to your homescreen, with tiles for the default apps such as Groups,
+Talk, and Terminal.
+
+### 5. Runtime Upgrades
+
+When new versions of the Urbit runtime (Vere) are available, you'll have to 
+shutdown your urbit and then run the `next` command in order to upgrade.
+```
+:dojo> |exit
+<pier>/.run next
+```
+Note that `<pier>` in this case is the folder that was created when you first booted
+your urbit. When you run this command your urbit will download the latest binary
+and place it inside the `.bin` directory inside your pier folder. You can then 
+start your ship with the following and you'll be on the latest runtime version:
+```
+<pier>/.run
+```
+> Linux users need to run this command in another terminal window to access
+> your urbit on port 80 every time you upgrade your runtime (otherwise it'll 
+> default to port 8080):
+>
+> ```shell
+> sudo apt-get install libcap2-bin
+> sudo setcap 'cap_net_bind_service=+ep' ./urbit
+> ```
+
+If you've been running Urbit for a while (from before runtime version 1.9) 
+and these `.run` commands don't work for you, it probably means you need to dock 
+your pier. You can do this with the following command:
```

Comment:
What is docking your pier? Is this documented anywhere (e.g. by running `./urbit dock` w/o arguments? It's the first question that popped out to me. Probably wouldn't hurt to have something somewhere that explains what this is doing.

Reply:
https://operators.urbit.org/manual/running/vere#dock - I can link the word dock to this.

Reply:
Perfect, exactly what I was looking for.